integration-test/src/test/java/org/sagebionetworks/ITDownloadStorageReportTest.java

```diff
@@ -0,0 +1,83 @@
+package org.sagebionetworks;
+
+import static org.aspectj.bridge.MessageUtil.fail;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.sagebionetworks.client.SynapseAdminClient;
+import org.sagebionetworks.client.SynapseAdminClientImpl;
+import org.sagebionetworks.client.SynapseClient;
+import org.sagebionetworks.client.SynapseClientImpl;
+import org.sagebionetworks.client.exceptions.SynapseException;
+import org.sagebionetworks.client.exceptions.SynapseResultNotReadyException;
+import org.sagebionetworks.repo.model.AuthorizationConstants;
+import org.sagebionetworks.repo.model.asynch.AsynchJobState;
+import org.sagebionetworks.repo.model.report.DownloadStorageReportResponse;
+import org.sagebionetworks.repo.model.report.StorageReportType;
+
+public class ITDownloadStorageReportTest {
+
+	private static SynapseAdminClient adminSynapse;
+	private static SynapseClient synapse;
+	private static Long userToDelete;
+	private static final long RETRY_TIME = 1000L;
+
+	@BeforeClass
+	public static void beforeClass() throws Exception {
+		StackConfiguration config = StackConfigurationSingleton.singleton();
+
+		// Create a user
+		adminSynapse = new SynapseAdminClientImpl();
+		SynapseClientHelper.setEndpoints(adminSynapse);
+		adminSynapse.setUsername(config.getMigrationAdminUsername());
+		adminSynapse.setApiKey(config.getMigrationAdminAPIKey());
+		adminSynapse.clearAllLocks();
+		synapse = new SynapseClientImpl();
+		userToDelete = SynapseClientHelper.createUser(adminSynapse, synapse);
+		adminSynapse.addTeamMember(AuthorizationConstants.BOOTSTRAP_PRINCIPAL.SYNAPSE_REPORT_GROUP.getPrincipalId().toString(), userToDelete.toString(), "https://www.synapse.org/#Team:", "https://www.synapse.org/#unsub:");
+
+		SynapseClientHelper.setEndpoints(synapse);
+	}
+
+	@AfterClass
+	public static void afterClass() throws Exception {
+		try { adminSynapse.deleteUser(userToDelete); } catch (SynapseException e) { }
+	}
+
+	@Test
+	public void generateReportAndGet() throws SynapseException, InterruptedException {
```

Comment:
Change this to just test an unauthorized user. We just need to make sure we got through the controller